stores/nvidia.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, gpu, locale="en_us", test=False, interval=5):
         adapter = TimeoutHTTPAdapter(
             max_retries=Retry(
```

Comment:
Maybe put max_retries as a parameter with this as the default value? That way it can be adjusted on a per store basis, if needed?

Reply:
Let me know if my latest changes align with what you've suggested here.